home/tests/import-export-data/bad_score_field.csv

```diff
@@ -0,0 +1,326 @@
+
+Here's the next one: 
+
+👤 *If I work hard, I will be successful.*",,,,,"Agree,Not sure,Disagree","0.0,1.0,0.0","success-agree,success-not-sure,success-disagree",work-question,",,"
+Placeholder_form_01,categorical_question,placeholder_form,mnch_onboarding_placeholder_form,v1.0,en,mnch_onboarding_placeholder_form_results_01,75,mnch_onboarding_placeholder_form_results_02,50,mnch_onboarding_placeholder_form_results_03,1,skip-result-page,Please choose an option that matches your answer,"How about this one?
+
+🌞 *I can get relevant health advice if and when I want it.*",,,,,"Agree,Not sure,Disagree","0.0,1.0,0.0","advice-agree,advice-not-sure,advice-disagree",advice-question,",,"
+Placeholder_form_01,categorical_question,placeholder_form,mnch_onboarding_placeholder_form,v1.0,en,mnch_onboarding_placeholder_form_results_01,75,mnch_onboarding_placeholder_form_results_02,50,mnch_onboarding_placeholder_form_results_03,1,skip-result-page,Please choose an option that matches your answer,"Thanks, and how about this statement:
+
+🌞 *What I do mainly depends on other people.*",,,,,"Agree,Not sure,Disagree","0.0,1.0,0.0","depend-agree,depend-not-sure,depend-disagree",dependence-question,",,"
+Placeholder_form_01,categorical_question,placeholder_form,mnch_onboarding_placeholder_form,v1.0,en,mnch_onboarding_placeholder_form_results_01,75,mnch_onboarding_placeholder_form_results_02,50,mnch_onboarding_placeholder_form_results_03,1,skip-result-page,Please choose an option that matches your answer,"Last one...
+
+🌞 *I have someone to talk to when I need it.*",,,,,"Yes,No,Skip this question","0.0,1.0,0.0","talk-yes,talk-no,talk-skip",talk-question,",,"
+Explainer Test,categorical_question,explainer-test,explainer-test,,en,assessment-result-high,5,assessment-result-moderate,3,assessment-result-low,0,,This is a generic error,What's your blood pressure,We need to know this to give you a proper risk assessment,Please reply with a number that matches your answer,,,"Below 120,120-130,Above 130","2.0,4.0,5.0","below-120,120-130,above-130",sem 001,",,"
+Explainer Test,age_question,explainer-test,explainer-test,,en,assessment-result-high,5,assessment-result-moderate,3,assessment-result-low,0,,This is a generic error,How old are you,It's important that we know your age for a proper assessment,Please reply with a valid age,,,,,,sem 002,
+Explainer Test,categorical_question,explainer-test,explainer-test,,en,assessment-result-high,5,assessment-result-moderate,3,assessment-result-low,0,,This is a generic error,What's your body temperature,,Please select an option that matches your answer,,,"Below 37C,Above 37C","1.0,5.0","below-37,above-37",sem 003,","
```

Comment:
Could we maybe use a smaller example, that just shows the issue with the score? It's very difficult to be able to see the bad example in this big CSV